LondonBritishLibrary/orient/BLorient539.xml

```diff
@@ -0,0 +1,324 @@
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS00001BL"/>
+                        <collection>Oriental</collection>
+                        <idno>BLorient 539</idno>
+                        <altIdentifier>
+                            <idno>Wright cat. CXVIII</idno>
+                        </altIdentifier>
+                        <altIdentifier>
+                            <idno>Wright 118</idno>
+                        </altIdentifier>
+                    </msIdentifier>
+                    
+                    <msContents>
+                        <summary/>
+                        
+                        <msItem xml:id="ms_i1">
+                            <title type="complete" ref="LIT2701Dawit"/>
+                            <textLang mainLang="gez"/>
+                            
+                            <msItem xml:id="ms_i1.1">
+                                <title type="complete" ref="LIT3985Mazmura"/>
+                                
+                                
+                                <msItem xml:id="ms_i1.1.1">
+                                    <title type="complete" ref="LIT3985Mazmura#Psalms"/>
+                                    
+                                    <msItem xml:id="ms_i1.1.1.1">
+                                        <title type="complete" ref="LIT4691Prayer"/>
+                                        <note>The single prayers of <ref type="work" corresp="LIT4691Prayer"/> probably occur at the beginning and after each book of
+                                            Psalms; the catalogue description only offers the analogy with <ref type="mss" corresp="BLorient537"/>.</note>
+                                    </msItem>
+                              </msItem>
+                                
+                                <msItem xml:id="ms_i1.1.2">
+                                    <locus from="160r"/>
+                                    <title type="complete" ref="LIT3985Mazmura#Canticles"/>
+                                </msItem>
+                                
+                                <msItem xml:id="ms_i1.1.3">
+                                    <locus from="175v"/>
+                                    <title type="complete" ref="LIT3985Mazmura#SongOfSongs"/>
+                                </msItem>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.2">
+                                <locus from="184v"/>
+                                <title type="complete">Prayers to Mary and to Christ</title>
```

Comment:
No ID here?

Reply:
No, I think that there is not enough information here to assign an ID.